engine/image-loader/t/tga-test.lisp

```diff
@@ -1,16 +1,17 @@
 (defpackage :lbge.test.tga
-  (:use :cl :rove :lbge.image-loader.image))
+  (:use :cl :rove :lbge.image-loader.image)
+  (:import-from :alexandria))
 
 (in-package :lbge.test.tga)
 
-(defvar *lbge-skip-test*)
-
 (defun instance-of-p (class obj)
   (eq class (type-of obj)))
 
 (deftest tga-test
-    (let ((image (lbge.image-loader.tga:tga "test-file.tga")))
+    (let ((image (lbge.image-loader.tga:tga "test-file.tga"))
+          (test-data (alexandria:read-file-into-byte-vector)))
```

Comment:
If you use `:import-from`, there's no need in specifying package name, you can use just `read-file-into-byte-vector`.